### PR TITLE
Web 5868 fix format transcript api

### PR DESCRIPTION
**Pull request by hislopzach**
https://invoca.atlassian.net/browse/WEB-5868
fix incorrect `transcript_format` value listed on the transcript api page. Some values were incorrectly listed as `agent_caller_block` and `agent_caller_conversation`, and have been fixed to display the correct values of `caller_agent_block` and `caller_agent_conversation`

update readme's `getting started` to:
* specify python version 
*  include desired pip version ([context for changes made](https://invoca.slack.com/archives/C023NRFGFFU/p1628208897114200?thread_ts=1628208391.113700&cid=C023NRFGFFU))

## Checklist

- [ ] Find the [Service owning team](https://docs.google.com/spreadsheets/d/1YF2wuepY5SZTpVEdT9gwhTPhpnDGZeVjywEwvbJS5TQ/edit#gid=0&fvid=1520238175) for these changes, and tag the team as "Reviewers" on this PR
- [ ] Test the documentation changes on readthedocs as a private branch
- [ ] If changing general content, have agreement on whether to apply to latest version or all versions (if all versions, provide links to the related PRs below)

### CODE REVIEW

**Comment by omeedrj** [comment]
I'll remove myself for now as a reviewer since this still needs a review from the owning team. I'll be watching the PR so once it's approved I can merge whenever it's ready to be published (on merge it is published immediately).

In the future you can mention in a comment or tag for a review the group: @Invoca/developer-docs-administrators

File: README.md
Comment (alexhornick-dt) [inline README.md:23]: Thank you! 😄

**Review by alexhornick-dt — APPROVED** [review]
Looks good to me! Cross-checked the new value to the hard-coded transcript formats to be safe, and it's correct. ✅

**Pull request merged**
